docs/user-guide/helm.md

```diff
@@ -199,6 +201,60 @@ You have to remember about `HELM_PLUGINS` environment property - this is require
 
 After that you have to use your custom image for ArgoCD installation.
 
+### Using `initContainers`
+Another option is to install Helm plugins via Kubernetes `initContainers`.
+Some users find this pattern preferable to maintaining their own version of the ArgoCD container image.
+
+Below is an example of how to add Helm plugins when installing ArgoCD with the [official ArgoCD helm chart](https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd):
+
+# helm-gcs plugin
+repoServer:
+  volumes:
+    - name: helm
+      emptyDir: {}
+    - name: gcloud
+      secret:
+        secretName: helm-credentials
+  volumeMounts:
+    - mountPath: /helm
+      name: helm
+    - mountPath: /gcloud
+      name: gcloud
+  env:
+    - name: HELM_DATA_HOME
+      value: /helm
+    - name: HELM_CACHE_HOME
+      value: /helm/cache
+    - name: HELM_CONFIG_HOME
+      value: /helm/config
+    - name: HELM_PLUGINS
+      value: /helm/plugins/
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: /gcloud/key.json
+  initContainers:
+    - name: install-helm-plugins
+      image: alpine/helm:3.6.3
+      volumeMounts:
+        - mountPath: /helm
+          name: helm
+        - mountPath: /gcloud
+          name: gcloud
+      env:
+        - name: HELM_DATA_HOME
+          value: /helm
+        - name: HELM_CACHE_HOME
+          value: /helm/cache
+        - name: HELM_CONFIG_HOME
+          value: /helm/config
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /gcloud/key.json
+      command: ["/bin/sh", "-c"]
+      args:
+        - apk --no-cache add curl;
+          helm plugin install https://github.com/hayorov/helm-gcs.git;
+          helm repo add bulder-shared gs://my-private-helm-gcs-repository;
+          chmod -R 777 $HELM_DATA_HOME;
```

Comment:
I think this block needs to be wrapped into triple-backticks as a code block in order to display correctly.

Reply:
whoops, my bad